recipe/build_win.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+export BUILD_PREFIX_SH=${BUILD_PREFIX//\\//}
+export BUILD_PREFIX_SH=/${BUILD_PREFIX_SH//:}
+
+# Patch Makeconf
+sed -i -e "s;g++;${BUILD_PREFIX_SH}/Library/bin/clang++.exe;g" $BUILD_PREFIX/Lib/R/etc/x64/Makeconf
+sed -i -e "s;nm;${BUILD_PREFIX_SH}/Library/bin/llvm-nm.exe;g" $BUILD_PREFIX/Lib/R/etc/x64/Makeconf
+sed -i -e "s;gnu++11;gnu++14;g" $BUILD_PREFIX/Lib/R/etc/x64/Makeconf
+
+# Patch Rcpp
```

Comment:
Is there a reference or references for why these patches are correct, or what they're doing? And now that you have a passing build, are you sure all of them are necessary?

Reply:
The Rcpp patch it is already upstreamed, the other ones are all hacky versions of what could be the correct approach. 

I have made this second PR in addition to #15 as I wanted to make sure I have only the necessary patches. Sadly, I need all of them

Reply:
I still have to find the correct way to override `Makeconf` without touching files in the user's home.